### ISSUE TITLE
Switching to new SUSYTools V7

### ISSUE THREAD

**Issue by amete** [susynt/SusyCommon#76]
Hi @dantrim, @gerbaudo, @ataffard,

There is a new ST beta version (V7 - current svn trunk) that is currently being tested as us being one of the testers. There a a couple of changes:

+ Switch to config files
+ Some interface changes
+ Reshuffling where decorations are being attached to objects:
``` 
- So in the new version all the methods are called in the correct order inside the GetObj() method, and > the user just has to

GetElectrons()
[...]
GetJets()
OverlapRemoval()

to get all the objects with the proper decorations.
(All the decorations methods are anyway still available for calls outside GetObj() )
```

I've just made a few modifications on our side so that SusyCommon compiles w/ this new ST, and commited a new brach (mc15_newGenST - c65db35c424223ccae1dfa2be2c82b5391afde23). I haven't yet tested it to see how things look on the practical side but that'll follow.

Best,
Serhan

**Comment by amete** [comment]
Hi guys,

So OK. I had the actual tests and there are a few things that need to be ironed out, but nothing on our side:

```
+ Conf. file is not actually configurable
```
==> Hardcoded in SUSYTools to get around
==> Discussed w/ Federico, will be fixed in SUSYTools

```
+ There is no electron isolation SF available for VeryLooseLH or LooseLH
```
==> Only load the MediumLH values to get around
==> Discussed w/ Federico, will be safeguarded in SUSYTools

```
+ TauSmearingTool_mc15 bails out w/ "No truth match information available. Please run TauTruthMatchingTool first." since "::truthParticleLink" doesn't exist. 
```

E-mail from TJ (27/8) says: 

```
"For now, we’re suggesting you set up the TauTruthMatchingTool and run it yourself before calling SUSYTools, but in the longer term this information should be added to the derivations (currently being worked on). "
```
==> According to Federico this is high prio. for the next derivation cache. So I'm not touching anything at the moment. 

So I would keep this branch as it is until the new SUSY cache is ready and the next gen. of SUSYTools is released. Then revisit these issues and make sure they are addressed. Then we can merge and continue other developments. 

Please let me know what you think.

Best,
Serhan

**Comment by amete** [comment]
BTW, as for the isolation SFs for various working points, this needs to be checked.

-s

**Comment by amete** [comment]
For our feedback, besides the technical ones, there is one thing that we can brainstorm:

```
Besides a general check that things still make sense, we would like to collect a bit of feedback on the new external config structure itself and in particular on what you think should be migrated from tool property to a simple configuration entry and vice-versa depending on your analyses needs.
```

Please let me know your input.

-s

**Comment by gerbaudo** [comment]
This is now being implemented on the `n0217-dev` branch.
Regarding the issues mentioned by @amete 
- config and `TauSmearingTool` fixed in 036e3f2
- missing electron scale factors for looseLH: fixed in c09d015

Closing this issue.
